app/services/import_actuals.rb

```diff
@@ -129,6 +137,15 @@ def initialize(row)
       @activity = @attributes.delete(:activity)
     end
 
+    # We must have precisely one value, and we will check if
+    # the other is empty when we validate the value it contains
+    def actual_class
+      return Actual if @row["Refund Value"].nil?
+      return Refund if @row["Actual Value"].nil? || @row["Actual Value"] == "0.00"
+      @errors[:value] = [@row["Actual Value"], I18n.t("importer.errors.actual.non_numeric_value")]
+      # @errors[:refund_value] = [@row["Refund Value"], I18n.t("importer.errors.actual.non_numeric_value")]
```

Comment:
Does this need to be an error message in its own right, or can we delete the commented line?

Reply:
:+1: